dts/arm/allwinner/sun8i-h3.dtsi

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2025 Muhammad Waleed Badar
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+
+#include <arm/armv7-a.dtsi>
+#include <zephyr/dt-bindings/interrupt-controller/arm-gic.h>
+
+/ {
+	interrupt-parent = <&gic>;
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			compatible = "arm,cortex-a7";
```

Comment:
this does not exist, the 3 bindings are:
`arm,cortex-a78`
`arm,cortex-a76`
`arm,cortex-a72`
So would need to use one or create one specifically for your SoC, which would have the clock frequency property in it

Reply:
Should have added that commit here so it could be merged with this, otherwise you'll have to wait enough time for people to review that and it to get merged, then rebase and update this PR and repeat the process